### PR TITLE
Several improvements (see description comment)

### PR DESCRIPTION
**Pull request by gerardishare**
- List correct possible role values and remove references to scheme owner
- Added getting started section for Service Consumer
- Copied changes from https://github.com/iSHAREScheme/Documentation/pull/17 to 2.0 branche
- Copied changes from https://github.com/iSHAREScheme/Documentation/pull/12 to 2.0 branche

### CODE REVIEW

File: service-consumer/getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+.. list-table:: Consuming iSHARE services: prerequisites
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - Step
+     - iSHARE Staging
+     - iSHARE Production
+   * - Obtain a valid identity (private key + certificate)
+     - `Click here <https://ca7.isharetest.net:8442/ejbca/ra/index.xhtml>`_ to request a test certificate. More information on :ref:`Get Test Certificate section<refGetTestCert>`.
+     - TBD
```
Comment (rajiv-ishare) [inline service-consumer/getting-started.rst:20]: what is to be determined here?

File: service-consumer/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+*Connection with the Authorization Registry*
+
+#. Get Access Token
+#. Use the Access Token to invoke the Delegation endpoint, to obtain Delegation Evidence
+
+*Connection with the Service Provider*
+
+#. Get Access Token
+#. Use Access Token and Delegation Evidence to consume service
```
Comment (rajiv-ishare) [inline service-consumer/getting-started.rst:87]: would be great to add (sequence) diagrams for the steps mentioned above for better readability

File: satellite/parties-id.rst [hunk 3]
```diff
@@ -29,7 +29,7 @@ Parameters
 
 ``certificate_subject_name``
     | **String**. Query parameter. *Optional*.
-    | subjectName as encoded in the X.509 certificate which corresponds with the party that is being requested from the Scheme Owner. Used by the Scheme Owner to match the certificate identifier. Subject name attributes may be in any order, but all of them must be included and separated by comma, if at least one subject attribute is missing - information won't be returned. Only returns info if combined with the valid ``eori`` associated to it.
+    | subjectName as encoded in the X.509 certificate which corresponds with the party that is being requested from the iSHARE Satellite. Used by the iSHARE Satellite to match the certificate identifier. Subject name attributes may be in any order, but all of them must be included and separated by comma, if at least one subject attribute is missing - information won't be returned. Only returns info if combined with the valid ``eori`` associated to it.
```
Comment (rajiv-ishare) [inline satellite/parties-id.rst:32]: MUST be replaced with:
subjectName as encoded in the X.509 certificate which corresponds with the party that is being requested from the iSHARE Satellite. Subject name attributes may be in any order, but all of them must be included and separated by comma, if at least one subject attribute is missing - information won't be returned.

**Review by rajiv-ishare — COMMENTED** [review]
added few comments

File: service-consumer/getting-started.rst (hunk 1 above)
Comment (gerardishare) [reply service-consumer/getting-started.rst:20]: Done, will follow in next version.

File: service-consumer/getting-started.rst (hunk 2 above)
Comment (gerardishare) [reply service-consumer/getting-started.rst:87]: Done, will follow in next version.

File: satellite/parties-id.rst (hunk 3 above)
Comment (gerardishare) [reply satellite/parties-id.rst:32]: Done, will follow in next version.

File: service-consumer/resources/service-consumer-scenario-3.png
Comment (rajiv-ishare) [inline service-consumer/resources/service-consumer-scenario-3.png:1]: The sequence of the capabilities and token endpoint call can be reversed for  call  from service consumer to service provider. As the parties endpoint will only return the capabilities endpoint and not token endpoint. The flow in the file now assumes that Service Consumer already has the token endpoint of Service Provider. In most idealistic manner, you could put the capabilities endpoint in a optional box and add that to current flow, where the call to capabilities endpoint after token endpoint is with access token and also in a box to depict that it is optional. In short: 1 SC -> capabilities -> SP
2 SC -> token -> SP
3 SC -> capabilities (with token) -> SP
4 SC -> Service -> SP

File: service-consumer/resources/service-consumer-scenario-4.png
Comment (rajiv-ishare) [inline service-consumer/resources/service-consumer-scenario-4.png:1]: General comment on all sequence diagrams, always number the steps, so its easy to refer to them.
I do not understand the steps after the first call to parties endpoint (from SC to satellite) what certificates is it testing? of Satellite?
Also, prechecks by service consumer is optional, so would be good to show that in optional box

**Review by rajiv-ishare — COMMENTED** [review]
some more changes!

File: service-consumer/resources/service-consumer-scenario-4.png
[the remark this answers is not in the document]
Comment (gerardishare) [reply service-consumer/resources/service-consumer-scenario-4.png:1]: Fixed in update

File: service-consumer/resources/service-consumer-scenario-3.png
Comment (gerardishare) [reply service-consumer/resources/service-consumer-scenario-3.png:1]: Fixed in update

**Review by rajiv-ishare — APPROVED** [review]
Seem ok

**Pull request merged**